controllers/release/release_adapter_test.go

```diff
@@ -602,6 +615,26 @@ var _ = Describe("Release Adapter", Ordered, func() {
 		})
 
 		It("tracks the binding if one is found", func() {
+			adapter.ctx = loader.GetMockedContext(ctx, []loader.MockData{
+				{
+					ContextKey: loader.SnapshotEnvironmentBindingContextKey,
+					Resource: &applicationapiv1alpha1.SnapshotEnvironmentBinding{
+						ObjectMeta: metav1.ObjectMeta{
```

Comment:
I don't think you need the ObjectMeta at all. The binding is loaded from the Release status.